src/index.tsx

```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```

Comment:
Is this coming from prettier, eslint or my editor?
It's weird to me to see the last item with a comma

Reply:
It's because in prettier it was configured `"trailingComma": "all"` as you suggested, but I think we should use `"trailingComma": "es5"` (the default value) to avoid that

Reply:
Indeed, I don't remember from where I got this prettier config, but trailingComma all is very weird.
es5 seems fine indeed 👍